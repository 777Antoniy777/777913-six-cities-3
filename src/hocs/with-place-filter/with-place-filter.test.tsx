import * as React from "react";
import * as renderer from "react-test-renderer";
import withPlaceFilter from "./with-place-filter";

interface MockComponentProps {
  children: React.ReactNode;
}

// set mocha data
const hockProps = {};

const MockComponent = (props: MockComponentProps) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const PlaceFilterWrappedHoc = withPlaceFilter(MockComponent);

it(`render withPlaceFilter`, () => {
  const tree = renderer.create(
      <PlaceFilterWrappedHoc
        props={hockProps}
        setDefaultOrderOffers={() => null}
        setLowToHighOrderOffers={() => null}
        setHighToLowOrderOffers={() => null}
        setTopRatedFirstOrderOffers={() => null}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
