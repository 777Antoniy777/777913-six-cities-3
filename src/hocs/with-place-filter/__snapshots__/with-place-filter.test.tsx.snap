// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withPlaceFilter 1`] = `<div />`;
